README.md

```diff
@@ -1,2 +1,196 @@
-# helm-operator
-Helm Operator for OpenEBS Installation
+# OpenEBS Helm Operator
+
+## Introduction
+
+- This is  a helm operator for openebs
+- The content of values.yaml will be placed as spec items in `OpenEBSInstallTemplate` CR 
+- The operator watches all namespaces. There is no need for helm client/tillerto be installed.
+  (the helm operator imports code from helm project)
+- Changes to spec of the CR will behave similar to values override via helm
+- All OpenEBS resources have ownerReference of `OpenEBSInstallTemplate` & are deleted upon removal of CR
+
+Example of OpenEBSInstallTemplate: 
+
+  ```yaml
+  apiVersion: openebs.io/v1alpha1
+kind: OpenEBSInstallTemplate
+metadata:
+  name: oebs 
+  namespace: openebs
+spec:
+  # Default values copied from <project_dir>/helm-charts/openebs/values.yaml
+  
+  # Default values for openebs.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+  
+  rbac:
+    # Specifies whether RBAC resources should be created
+    create: true
+  
+  serviceAccount:
+    create: true
+    name:
+
+  release:
+    # "openebs.io/version" label for control plane components
+    version: "1.0.0"
+  
+  image:
+    pullPolicy: IfNotPresent
+  
+  apiserver:
```

Comment:
How do we specify if this is a deployment vs. a statefulset

Reply:
Please ignore this comment if this entire file corresponds to helm template values only

Reply:
Yes...it corresponds to existing template values